src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilter.java

```diff
@@ -89,6 +89,9 @@
  * VariableDeclarationUsageDistance
  * </li>
  * </ul>
+ * <p>Reason for RegexpSinglelineJava and Regexp support unavailability is at
```

Comment:
as I requested in https://github.com/checkstyle/checkstyle/issues/7759#issuecomment-605525287

please update lines:
```
 * Regexp
 * </li>
 * <li>
 * RegexpSinglelineJava
 * </li>
 * <li>
```

to:
```
 * Regexp (reason is at <a href="https://github.com/checkstyle/checkstyle/issues/7759#issuecomment-605525287"> #7759</a>)
 * </li>
 * <li>
 * RegexpSinglelineJava (reason is at <a href="https://github.com/checkstyle/checkstyle/issues/7759#issuecomment-605525287"> #7759</a>)
 * </li>
 * <li>
```

the same for `config_filters.xml`.

Reply:
done

Reply:
I have posted the website image in the first post